sklearn/mixture/dpgmm.py

```diff
@@ -501,10 +501,10 @@ def fit(self, X):
             List of n_features-dimensional data points.  Each row
             corresponds to a single data point.
         """
-        self.random_state = check_random_state(self.random_state)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
Oh ,right, I got it. Btw, do you think it is ok to modify it if a RandomState object was passed? (And do you want to open an issue for the test?)